### ISSUE TITLE
SUGGESTION: Simpler R package instructions

### ISSUE THREAD

**Issue by HenrikBengtsson** [etal/cnvkit#16]
You can install both Bioconductor and CRAN package using `biocLite()`, so it's enough to do:

```r
source("http://bioconductor.org/biocLite.R")
biocLite(c("PSCBS", "matrixStats", "cghFLasso"))
```

Further, since [PSCBS](http://cran.r-project.org/web/packages/PSCBS/index.html) imports [matrixStats](http://cran.r-project.org/web/packages/matrixStats/index.html), the latter is automatically installed, so it's even enough to do:
```r
source("http://bioconductor.org/biocLite.R")
biocLite(c("PSCBS", "cghFLasso"))
```

**Comment by HenrikBengtsson** [comment]
BTW, I've might have mentioned in the past (offline) that I have an even easier way installation, namely:
```r
source("http://callr.org/install#PSCBS,cghFLasso")
```
This http://callr.org approach still have low "mileage" though.

**Comment by etal** [comment]
Thanks! I update the docs to show the biocLite route.

Is the callr.org server running someplace with good uptime? Can you suggest a one-line shell command that would do the callr.org installation? That would simplify automated installations a bit.

**Comment by HenrikBengtsson** [comment]
Yes, callr.org has good up time.  Also, there is actually nothing running on that server; all that happens is that R downloads an R script '[install](http://callr.org/install)', which via "self-reflection" identifies which  packages to update from the URL.  In other words, it's also very light-weight on the callr.org web server.

To install from the shell/command line:
```sh
Rscript -e "source('http://callr.org/install#PSCBS,cghFLasso')"
```
I've verified that this works on Linux, OS X, and Windows.  It's important the the quotation marks are exactly as above.

(I didn't though about this use case when I designed it, so there are some output/message that are intended for interactive R sessions, but it works. I've added it to the to-do list to [make the above non-interactive call a bit more silent](https://github.com/callr-org/website/issues/4).
